istio-proxy/istio-proxy.spec

```diff
@@ -40,44 +39,62 @@ BuildRequires:  gcc
 BuildRequires:  gcc-c++
 BuildRequires:  make
 BuildRequires:  patch
-BuildRequires:  ksh
 BuildRequires:  xz
+BuildRequires:  git
 BuildRequires:  golang
 BuildRequires:  automake
 BuildRequires:  python3
 BuildRequires:  cmake3
 BuildRequires:  openssl
 BuildRequires:  openssl-devel
 BuildRequires:  libatomic
+BuildRequires:  platform-python-devel
 
-Source0:        istio-proxy.%{checksum}.tar.gz
-Source1:        build.sh
-Source2:        test.sh
-Source3:        fetch.sh
-Source4:        common.sh
+Source0:        proxy-%{git_commit}.tar.gz
+Source10:       gn-20200319.tar.gz
 
 %description
 The Istio Proxy is a microservice proxy that can be used on the client and server side, and forms a microservice mesh. The Proxy supports a large number of features.
 
-########### istio-proxy ###############
-%package istio-proxy
-Summary:  The istio envoy proxy
+%prep
+%setup -q -n proxy-%{git_commit}
+%setup -q -D -T -n proxy-%{git_commit} -a 10
 
-%description istio-proxy
-The Istio Proxy is a microservice proxy that can be used on the client and server side, and forms a microservice mesh. The Proxy supports a large number of features.
+%build
+# BEGIN GN FIXME: Move GN to its own package?
+pushd gn
```

Comment:
not sure where this is coming from? it's not downloaded by `update.sh`

Reply:
This is needed for building multi-arch; I brought it here from product in a attempt to have equal .specs. I've just removed it as we are not building multi-arch for community (copr does not even support the other arches).